README.rst

```diff
@@ -31,16 +31,17 @@ Or, if in a conda-based environment, with conda from the conda-forge channel:
 
     conda install -c conda-forge pykdtree
     
-Note that by default these packages are only built with OpenMP for linux platforms.
+Note that by default these packages (the binary wheels on PyPI and the binary
+package on conda-forge) are only built with OpenMP for linux platforms.
```

Comment:
Isn't OpenMP also used by default for conda builds on macOS? It may be using clang with libomp aliased as libgomp (macports provides such an alias but homebrew does not), or using gcc.

Reply:
Hhhmm apparently not:

https://github.com/conda-forge/pykdtree-feedstock/blob/7a449f55ffbae350e6f365b9d0bb1ad80a8413ed/recipe/meta.yaml#L14-L16

So only linux. I wonder if I did my own testing with openmp from conda-forge and decided it could work on conda-forge, but the recipe was never updated on conda-forge to attempt it.